app/objects/secondclass/c_link.py

```diff
@@ -46,16 +48,17 @@ def fix_ability(self, link, **_):
             link['ability'] = link_input.schema.dump(link_input)
         return link
 
+    @ma.pre_load()
+    def fix_executor(self, link, **_):
+        if 'executor' in link and isinstance(link['executor'], Executor):
+            link_input = link.pop('executor')
```

Comment:
Should this be `executor_input`?

Reply:
Changed to `executor`. Also updated the `pre_load()` for abilities to `ability`.